server/src/test/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinatorTest.java

```diff
@@ -848,6 +848,28 @@ public void testDeleteDataSourceMetadata() throws IOException
     Assert.assertNull("getDataSourceMetadataNullAfterDelete", coordinator.retrieveDataSourceMetadata("fooDataSource"));
   }
 
+  @Test
+  public void testDeleteSegments()
+  {
+    Exception e = null;
+    try {
+      coordinator.deleteSegments(SEGMENTS);
+    }
+    catch (Exception ex) {
+      e = ex;
+    }
+    Assert.assertNull(e);
```

Comment:
If an exception is thrown, the test will fail. We don't need to catch the exception and validate that it's null

```suggestion
    coordinator.deleteSegments(SEGMENTS);
```

Reply:
Thanks for your review! Changed.